ts/main.ts

```diff
@@ -51,5 +51,111 @@ $formInputs.addEventListener('submit', (event: Event) => {
 
   $imageSubmit.setAttribute('src', '/images/placeholder-image-square.jpg');
 
+  renderEntry(obj);
+  viewSwap('entries');
+  toggleNoEntries();
+
+  if (data.entries.length > 0) {
+    toggleNoEntries();
+  }
   $formInputs.reset();
 });
+
+function renderEntry(entry: Entry): HTMLLIElement {
+  const $objListItem = document.createElement('li');
+  $objListItem.classList.add('row');
+
+  const $columnHalfImage = document.createElement('div');
+  $columnHalfImage.classList.add('column-half');
+
+  $objListItem.append($columnHalfImage);
+
+  const $imgUrl = document.createElement('img');
+  $imgUrl.src = entry.photo;
+  $imgUrl.classList.add('image-submit');
+
+  $columnHalfImage.append($imgUrl);
+
+  const $columnHalfText = document.createElement('div');
+  $columnHalfText.classList.add('column-half');
+
+  $objListItem.append($columnHalfText);
+
+  const $title = document.createElement('h2');
+  $title.textContent = entry.title;
+  $title.classList.add('input-box');
+
+  $columnHalfText.append($title);
+
+  const $note = document.createElement('p');
+  $note.textContent = entry.note;
+  $note.classList.add('box-notes');
+
+  $columnHalfText.append($note);
+  if (!$ul) throw new Error('The $ul query failed');
+
+  $ul.prepend($objListItem);
```

Comment:
You should not be appending to `$ul` inside of `renderEntry`. You `return` the outtermost element, and do the appending where you call the function